src/Model/Tag.php

```diff
@@ -243,28 +243,32 @@ public static function getID(string $name, string $url = '', int $type = null)
 			return $tid;
 		}
 
+		// Also log type and tag id
+		$fields['type'] = $type;
+		$fields['tid'] = $tid;
```

Comment:
At this point `$tid` is empty, not sure it's relevant to include it.

Reply:
It was about the variable's type. Should I to a `gettype()` instead?

Reply:
I don't think so, it's the result of `mysql_insert_id()` which has predictable return value types.